### DISCUSSION TITLE
HTML Anchor Support

### DISCUSSION THREAD

**Discussion by masteramyx** [GitJournal/GitJournal#692] [Feature Requests]
What I typically see is

[Chapter 1](#ch-1) ->  ([Chapter 1]*(ch-1)) No asterisk

and then somewhere further down

<a* id="ch-1">CHAPTER 1</a*>.      (w/o the asterisk)

This would be nice to have for jumping inside a document.

**Comment by vHanda** [comment]
Note to self: Instead of making the current markdown renderer understand the concept of html, maybe it would be just easier to switch to an html based renderer when there is html or make this configurable.

https://github.com/Sub6Resources/flutter_html

**Comment by nonsleepr** [comment]
I think GitHub [does it better](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#section-links).

The link doesn't describe how it works but the idea is that each heading automatically generates an anchor in `lowercase-kebab-style`.

So, it would like like this:

```markdown
Go to [Chapter 1](#chapter-1-hello-world).

Lorem ipsum dolor

## Chapter 1: Hello World
```

**Comment by switzel** [comment]
I would also very much like to have anchors, preferably the GitHub flavor.
My current reason is that it would allow me to have a note with several lists with headings (say fruit, dairy, beverages) and then add a table of contents at the beginning to quickly jump to one of the lists.
